app/models/concerns/tagable.rb

```diff
@@ -6,6 +6,17 @@ module Tagable
     has_many :tags, through: :taggings
   end
 
+  TAGABLE_CLASSES = [Entity, List, Relationship]
+
+  # sorts a list of tagables in descending order of relationships to tagables w/ same tag
+  def self.sort_by_related_tagables(tagables)
+    tagables
+  end
+
+  def self.page_param_of(klass_name)
```

Comment:
This can become a regular method:

``` rb
def tagable_page_param
    (self.class.name.downcase + "_page").to_sym
  end
```

Reply:
way better.

Reply:
proposal: pick this up when playing #306